### ISSUE TITLE
Property with name class generates invalid Java code

### ISSUE THREAD

**Issue by joellp** [openapi-processor/openapi-processor-base#29]
When having properties with a name similar to one of the Java keywords it will generate Java code that is invalid. 

Example:

```json
{
  "openapi": "3.0.1",
  "components": {
    "schemas": {
      "Fare": {
        "type": "object",
        "properties": {
          "class": {
            "allOf": [
              {
                "$ref": "#/components/schemas/FareClass"
              }
            ],
            "description": "Class for which this fare is valid.",
            "xml": {
              "name": "Class"
            },
            "example": "First"
          }
        }
      }
    }
  }
}
```

This wil generate:

```java
@Generated(value = "openapi-processor-micronaut", version = "2023.2", date = "2023-06-13T08:59:00.859252900+02:00")
public class Fare {   

    @JsonProperty("class")
    private FareClass class; // This does not compile
    
    public FareClass getClass() {
        return class; // This does not compile
    }

    public void setClass(FareClass class) {
        this.class = class; // This does not compile
    }    

}
```

Is there a way to fix this?

**Comment by joellp** [comment]
Unfortunately I cannot change the OpenAPI yaml, because it is from an external service. So changing the properties is not possible.

Please note that this happens for all the Java keywords.

**Comment by hauner** [comment]
right, it should avoid all keywords.

What I didn't suggest, was excluding the endpoint (https://openapiprocessor.io/spring/2023.2/mapping/endpoint.html#_excluding_endpoints) this would be an option to implement it manually.

Problem with this is, that it still generates the broken interface/classes. One would need to avoid compilation of them...

**Comment by hauner** [comment]
fixed in 2023.2.1, https://github.com/openapi-processor/openapi-processor-base/releases/tag/v2023.2.1

just published it, it will take about an hour before it is available on maven central.

**Comment by joellp** [comment]
Thanks for the fix! The code does not yet compile. It still generates the method as `getClass` which is not allowed:

```java
public FareClass getClass() {  // 'getClass()' cannot override 'getClass()' in 'java.lang.Object'; overridden method is final
    return aClass;
}
```

Do you want me to open a new issue?

**Comment by hauner** [comment]
there is only one `get` and no `set` in Object, so it is just `getClass`.

**Comment by hauner** [comment]
next try :-)

fixed in 2023.2.2, https://github.com/openapi-processor/openapi-processor-base/releases/tag/v2023.2.2
